### PR TITLE
Fixed showing additional items for Service, fixed the REST logo

### PR DESCRIPTION
**Pull request by hhovsepy**
Issue https://github.com/kiali/kiali/issues/6665

Fixed the "REST API" logo.
Fixed Additional information showing for ServiceDetails page.

How to test it:
Make sure Kiali CR contains (by default contains, kiali configmap needs this):
```
additional_display_details:
  - annotation: kiali.io/api-spec
    icon_annotation: kiali.io/api-type
    title: API Documentation
```
Apply annotations into any of services:
```
kubectl annotate service kiali -n istio-system kiali.io/api-type=rest --overwrite
kubectl annotate service kiali -n istio-system kiali.io/api-spec=https://petstore.swagger.io/v2/swagger.json
```
Reload the Service list and details pages.
For testing in Workoads, the annotations need to be applied on workload template.

Screenshot from 2023-11-30 15-06-38
Screenshot from 2023-11-30 13-04-28
Screenshot from 2023-11-30 13-04-11
Screenshot from 2023-11-30 13-03-55

### CODE REVIEW

**Comment by hhovsepy** [comment]
Required helm PR https://github.com/kiali/helm-charts/pull/236

File: frontend/cypress/integration/common/services.ts [hunk 1]
```diff
@@ -68,6 +68,12 @@ When('user filters for label {string}', (label: string) => {
   cy.get('input[aria-label="filter_input_label_key"]').type(`${label}{enter}`);
 });
 
+
+When('user applies annotations', () => {
```
Comment (ferhoyos) [inline frontend/cypress/integration/common/services.ts:72]: I would specify in the title what type of annotations I am applying (for example: `user applies kiali api annotations`)

**Review by ferhoyos — APPROVED** [review]
LGTM, api and runtimes logo appears. Just two little comments that do not block the approval. 

I have noticed that there are some disalignments in the UI, but this can be considered out of scope of this PR and to be fixed in a separate issue:
image

**Comment by hhovsepy** [comment]
thank you @ferhoyos , comments approached

**Pull request merged**